roles/haproxy/templates/haproxy.cfg.j2

```diff
@@ -21,7 +21,11 @@ defaults
 
 listen stats
     mode http
+{% if cluster_vip is defined and cluster_vip | length > 0 %}
+    bind {{ cluster_vip }}:{{ haproxy_listen_port.stats }}
+{% else %}
     bind {{ hostvars[inventory_hostname]['inventory_hostname'] }}:{{ haproxy_listen_port.stats }}
+{% endif %}
```

Comment:
```suggestion
{% if cluster_vip is defined and cluster_vip | length > 0 %}
    bind {{ cluster_vip }}:{{ haproxy_listen_port.stats }}
{% endif %}
    bind {{ hostvars[inventory_hostname]['inventory_hostname'] }}:{{ haproxy_listen_port.stats }}
```

I suggest leaving listening to the local address, but also adding the possibility of listening to VIP

this will allow us to check the statistics for the host we need